handlers/warp/stats/stats.go

```diff
@@ -0,0 +1,95 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package stats
+
+import (
+	"sync"
+	"time"
+
+	"github.com/ava-labs/subnet-evm/metrics"
+)
+
+var (
+	_ SignatureRequestHandlerStats = (*handlerStats)(nil)
```

Comment:
Trying to figure out difference between `(*handlerStats)(nil)` and `&handlerStats{}`, is it the difference between nil and a default struct value?

Should we always use `(*struct)(nil)` i'm thinking for `MockHandlerStats` etc

Reply:
They're both used to ensure that the struct implements the interface.

I've switched to using `(*type)(nil)`. It doesn't make much of a difference and tbh we should be consistent, but currently use both in different places.

Here's one article: https://splice.com/blog/golang-verify-type-implements-interface-compile-time/.